device/bgp/routepathattributes.yaml

```diff
@@ -22,3 +22,19 @@ components:
               x-field-uid: 1
             egp:
               x-field-uid: 2
+            incomplete:
+              x-field-uid: 3
+        include_local_preference:
+          description: >-
+            BGP local preference attribute sent to internal peers to indicate the degree of preference 
+            for externally learned routes. It should be enabled only for internal peers.
+          type: boolean
+          default: true
+          x-field-uid: 3
+        local_preference:
+          description: >-
+            Local Preference attribute, it is used for iBGP peers to select the path for the traffic leaving the AS.
```

Comment:
Value to be set in the Local Preference attribute if include_local_preference is set to true. It is used for iBGP peers ...